Makefile

```diff
@@ -146,6 +146,10 @@ add-license-headers: $(GO_ADD_LICENSE)
 check-generate:
 	@hack/check-generate.sh $(REPO_ROOT)
 
+.PHONY: check-plutono-dashboards
+check-plutono-dashboards:
+	@./hack/check-plutono-dashboards.sh
+
 .PHONY: check
 check: $(GO_ADD_LICENSE) $(GOIMPORTS) $(GOLANGCI_LINT) $(HELM) $(IMPORT_BOSS) $(LOGCHECK) $(GOMEGACHECK) $(YQ)
```

Comment:
```suggestion
check: $(GO_ADD_LICENSE) $(GOIMPORTS) $(GOLANGCI_LINT) $(HELM) $(IMPORT_BOSS) $(LOGCHECK) $(GOMEGACHECK) $(YQ) $(PLUTONO_DASHBOARD_CHECK)
```

Reply:
Already added line `@hack/check-plutono-dashboards.sh` to run the shell directly for the check.

Reply:
Then we still need this to make sure the go binary is "installed", don't we?